src/com/company/Main.java

```diff
@@ -8,68 +8,53 @@
 
 
 public class Main {
```

Comment:
Code seems ok now. Fix the file formatting. The indentation and spaces are all wrong. There is a menu option available in IntelliJ

Code -> Reformat File

Reply:
done with File formatting